ratelimit/backends/simple.py

```diff
@@ -0,0 +1,108 @@
+import asyncio
+import time
+from collections import defaultdict
+from threading import Lock
+from typing import Dict, List, Optional
+
+from ..rule import Rule
+from . import BaseBackend
+
+lock = Lock()
+
+
+def synchronized(func):
```

Comment:
I prefer to put `Lock` in the instance, one storage structure corresponds to one lock.

Reply:
OK